mitre-attack/operations.py

```diff
@@ -99,13 +102,16 @@ def get_mitre_data(config, params):
         # force ingest, disregard version
         tactics = query_source(mem_source, tactics_filter)
         groups = query_source(mem_source, groups_filter)
+        techniques_all = query_source(mem_source, techniques_all_filter)
         techniques = query_source(mem_source, techniques_filter)
         sub_techniques = query_source(mem_source, sub_techniques_filter)
         mitigations = query_source(mem_source, mitigations_filter)
         software_malware = query_source(mem_source, malware_filter)
         software_tools = query_source(mem_source, tools_filter)
         relationships = query_source(mem_source, relationships_filter)
 
+    techniques = combine_techniques(techniques_all, techniques)
```

Comment:
instead of combine, can we get all and the exclude the ones with subtechnique 'True' in this function ? will save us one query and some memory